packages/api/tests/endpoints/collections/delete-collection.js

```diff
@@ -233,7 +232,7 @@ test.serial('DELETE successfully deletes if collection exists in Elasticsearch b
     esCollectionClient,
     testKnex,
   } = t.context;
-  const testCollection = fakeCollectionFactory();
+  const testCollection = fakeCollectionRecordFactory();
```

Comment:
Is this right?   I thought ES indexed API collections.

Reply:
[`860560d` (#2533)](https://github.com/nasa/cumulus/pull/2533/commits/860560d683e03df21efed31345dd3d6eb459886f)